src/styles/index.scss

```diff
@@ -1,3 +1,137 @@
+$titleColor: #060b35;
+$productColor: #616070;
+$background-button-color: #00acdc;
+$button-color: #fff;
+
+html {
+  font-family: "Roboto", sans-serif;
+  color: $productColor;
+  font-size: 10px;
+  font-weight: 400;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+
+  width: 200px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    margin: 0 0 5px;
+    color: $titleColor;
+  }
+
+  &__code {
+    margin-bottom: 8px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .stars {
+    display: flex;
+  }
+
+  .stars__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-image: url(\images\star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  .stars__star:nth-child(-n+1),
+  .stars__star:nth-child(-n+2),
+  .stars__star:nth-child(-n+3),
+  .stars__star:nth-child(-n+4) {
+    background-image: url(\images\star-active.svg);
+  }
```

Comment:
move this to another file as different block

Reply:
not fixed